Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,10 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  // E5:
+  return arr.filter((value, index, array) => array.indexOf(value) === index);
+  // E6:
+  // console.log([...new Set(arr)]);
```

Comment:
You ES6 solution would have worked too (and passed the test) if you returned it rather then used console.log:

```js
return [...new Set(arr)];
```